ui/webui/src/components/storage/MountPointMapping.jsx

```diff
@@ -345,8 +345,8 @@ const MountPointMappingContent = ({ deviceData, partitioningData, dispatch, idPr
                 return {
                     "device-spec": cockpit.variant("s", row["device-spec"]),
                     "format-type": cockpit.variant("s", row["format-type"]),
-                    "mount-point": cockpit.variant("s", newRequest !== undefined ? newRequest["mount-point"] : row["mount-point"]),
```

Comment:
This commit I split it out in https://github.com/rhinstaller/anaconda/pull/5004/commits as well because i needed it. 

So one of the two PRs will need a clean rebase.